components/Location.js

```diff
@@ -32,7 +30,7 @@ class Location extends React.Component {
           <Container font-size="1.7rem" margin="5px 0px">
             {name}
           </Container>
-          <div className="address" style={{ color: colors.darkgray }}>
+          <div className="address" style={{ color: props => props.theme.colors.darkgray }}>
```

Comment:
I'd be surprised if this works:
- `div` is not wrapped in styled-components, so it doesn't have access to the theme
- `style` prop does not accept functions as values: https://fr.reactjs.org/docs/dom-elements.html#style 
- `darkgray` doesn't exist in `lib/theme/colors.js`

Did you try running the frontend to see what were the effects of your changes?

Reply:
I did run the frontend locally and it seems to work fine with no errors or warnings but I'm not sure where I would observe the changes.

- Should I create a new *StyledDiv.js* to change the color, then? I see that other simple styled-components are defined within the file directly, `LocationSection`, for example in *Location.js* but 4/5 references to lib/constants/colors.js are for the same color so I don't want to duplicate.
- Ok; Still learning.
- Ah, ok; I will create one. For some reason I wasn't finding the *colors.js* file.

Reply:
> Should I create a new StyledDiv.js to change the color, then?
That's one option, but for these simple cases we often rely on some generic styled components that already have support for theming. Here for example you could do:

```jsx
<Container color="black.600">
```

> Ah, ok; I will create one. For some reason I wasn't finding the colors.js file.

No need to create a new one, as per the issue description we should replace them with existing colors from the design system. For this specific one, I think the closest color is `black.600`, but feel free to use `black.700` if it's not dark enough.

Reply:
I made the changes. Tests pass but on running the front-end I'm getting the following error when creating a new org or accessing my existing:

```
./components/collective-page/BudgetStats.js:4:0
Module not found: Can't resolve '@styled-icons/ionicons-solid/Expand'
  2 | import PropTypes from 'prop-types';
  3 | import { Calendar } from '@styled-icons/feather/Calendar';
> 4 | import { Expand } from '@styled-icons/ionicons-solid/Expand';
  5 | import { ShowChart } from '@styled-icons/material/ShowChart';
  6 | import { FormattedMessage } from 'react-intl';
  7 | import styled, { css } from 'styled-components';
```

I'm unfamiliar with the @styled-icons reference but assuming it stems from the changes I made, most likely it's caused by search.js

I added: `import defaultColors from '../lib/theme/colors';`
and:       `${defaultColors.blue}`

I haven't pushed any changes since it's not solved but if it would be helpful, let me know.

Reply:
@ericyoungson : I don't think such a module problem is caused by your code unless you have removed a node module from `package.json`. Could you try `npm install` to sync the npm modules and see if that resolves the issue. If not feel free to post on slack and I can help you with it. 😄

Reply:
As @SudharakaP said, an `npm install` should resolve your problem.

Importing `'../lib/theme/colors'` is not a good option, the theme needs to be accessed from the props as it can differ from the default one. I mentioned the solution above, either wrap your component with styled-components or use an existing one like `Container`.

Reply:
> @ericyoungson : I don't think such a module problem is caused by your code unless you have removed a node module from `package.json`. Could you try `npm install` to sync the npm modules and see if that resolves the issue. If not feel free to post on slack and I can help you with it. 😄

Yes! That did it.

Reply:
> As @SudharakaP said, an `npm install` should resolve your problem.
> 
> Importing `'../lib/theme/colors'` is not a good option, the theme needs to be accessed from the props as it can differ from the default one. I mentioned the solution above, either wrap your component with styled-components or use an existing one like `Container`.

This reference appears to be a parameter within a styled-component definition rather than an HTML <div>
See [comment](https://github.com/opencollective/opencollective-frontend/pull/5887/files#r576889258)